source-map-support.js

```diff
@@ -21,6 +21,10 @@ function isInBrowser() {
   return ((typeof window !== 'undefined') && (typeof XMLHttpRequest === 'function'));
 }
 
+function hasGlobalProcess() {
```

Comment:
Hmm, might be a good idea. Do you have any suggestions? I don't really know what else to call it.